### ISSUE TITLE
Add reportlet to check BOLD-to-standard registration

### ISSUE THREAD

**Issue by oesteban** [nipreps/fmriprep#2445]
Generate a reportlet showing the resampling of bold into standard spaces to allow better checking the normalization and avoid regression of issues such as #2307.

Since reports are pretty busy already, it would be nice to have some flag for more comprehensive reporting on the CLI, or check whether the HTML `<details>` tag is implemented in browsers with some sort of lazy loading.

**Comment by effigies** [comment]
`<details>` has [been around](https://www.w3schools.com/TAGS/tag_details.asp) for a while and is well-supported.

**Comment by oesteban** [comment]
Yup, the question is whether the reportlet is going to be preloaded or the
download skipped until details is unfolded.

**Comment by effigies** [comment]
Ah, I misread. We can try adding the [`<img loading="lazy">`](https://www.w3schools.com/tags/att_img_loading.asp) attribute, which is supported in everything but Safari.

Testing in Chrome and Firefox (via `python -m http.server` so I can see network hits), it does delay loading until the summary is clicked.

I would suggest we add `loading="lazy"` to all image tags, whether we hide anything behind details.

**Comment by ly6ustc** [comment]
> Generate a reportlet showing the resampling of bold into standard spaces to allow better checking the normalization and avoid regression of issues such as #2307.
> 
> Since reports are pretty busy already, it would be nice to have some flag for more comprehensive reporting on the CLI, or check whether the HTML `<details>` tag is implemented in browsers with some sort of lazy loading.

Hi,  
Recently, I have used the fmriprep 21.0.0rc2. But it appears that there is not this report of checking normalization. May it take more time for this achievement? What's more, one of the attractions of fmriprep tool is preprocessing data in surface space, so could we add another report to check BOLD-to-standard registration of surface? Or how do we check the normalization results in surface space such as fsaverage6, fsLR91k, etc.? Also proposed in #2410.
